datastore.go

```diff
@@ -29,13 +29,26 @@ type Datastore struct {
 var _ ds.Datastore = (*Datastore)(nil)
 var _ ds.Batching = (*Datastore)(nil)
 
+var defaultSplit = func(a []byte) int {
+	return len(a)
+}
```

Comment:
In theory, we could allow for custom splits as our split is the most conservative (everything is the primary key). So if the user passes a custom split, we can call it instead.